h2o-py/h2o/h2o.py

```diff
@@ -1257,6 +1259,102 @@ def make_metrics(predicted, actual, domain=None, distribution=None):
               data={"domain": domain, "distribution": distribution})
     return res["model_metrics"]
 
+gen_model_file_name = "h2o-genmodel.jar"
+h2o_predictor_class = "water.util.H2OPredictor"
+
+def find_file(name, path):
+    """
+    Helper function for predict_json() function to check if both MOJO and h2o-genmodel.jar are available
```

Comment:
this method is not MOJO specific, the documentation is misleading IMO

Reply:
moved to shared_utils.py